### ISSUE TITLE
@everyone - Here are better Instructions on how to use the Twitch Chatlog tool.

### ISSUE THREAD

**Issue by gaveitatry02b** [freaktechnik/twitch-chatlog#106]
**INTRODUCTION:**

For anyone reading this, the GitHub developer "freaktechnik" banned me for using swear words.  But instead of thinking "oh, gaveitatry02 is a bad guy", the bad guy here is most definitely "freaktechnik".

1.) For the average Twitch user who has no experience with nodejs, python, etc., if they come to this GitHub they won't have a clue on what to do.  Why?  Because the Instructions are incomplete and useless.  It just says to type "npm ...".  Not everyone knows what a npm command is or where to type it.
2.) This "freaktechnik" jerk will discriminate against you if you aren't a node.js user.  He will act like you are too stupid to use his tool and that you must be a nodejs user to make any use of it.  "For a node.js audience only".  I am quoting his words.  The very second that he mentioned "requires node.js", I went and downloaded node.js and 20 seconds later I was using his tool with absolutely no difficulties whatsoever.  So you DON'T need to be a previous node.js user to use his tool.  The problem here is that he never wrote in his Instructions "this tool requires that you download node.js".  He lacks the consideration to take 1-2 minutes of his life to write better instructions, and he thinks that he is better than the people who are not part of the "node.js audience".  He can't be bothered to make more user friendly instructions for them.
3.) I have since amended his worthless Instructions and have emailed them to him.  Guess what?  It's been three days and he still hasn't updated his Instructions.  He probably never will.  I could come back three years from now, and he will still have worthless instructions that won't help anyone who isn't already a node.js user.  And the sad part is that all he has to do is copy and paste the amended instructions that I sent to him in an email.  That is how little he cares, and how little consideration he has for human beings in general.

So I'm going to go ahead and share my better Instructions with you now.

**INSTRUCTIONS:**

**STEP 01** - Click the green 'Clone or download' button and download this GitHub repository.  It is a fast download and the folder size is very small.  After Step 04, you can delete this repository folder.

**STEP 02** - Download node.js.  The approximate size is 75 MB.  https://nodejs.org

**STEP 03** - Once you start the node.js installation process, there will be a checkbox beside the words "Automatically install the necessary tools.  Note that this will also install Chocolatey.  The script will pop-up in a new window after the installation completes."  Do NOT put a checkmark here.  You don't need this stuff to use the Twitch Chatlog tool.  It will just make the installation process take a lot longer, install a dozen things that you don't need or want, and will take up a lot more space on your HDD.

**STEP 04** - After node.js is installed, start up the node.js and type "npm install -g twitch-chatlog" without the quotation marks in the window and press Enter.  This will install the Twitch Chatlog tool.  After it is done installing, you can go ahead and delete the repository folder that you downloaded in Step 01.

**STEP 05** - You can now go look at the "Usage" section on his GitHub page and figure out what you need to type to pull up a Twitch chat log.  For me, I typed "twitch-chatlog -l 0 XXXXXXXXXX".  The "-l 0" part was important for me because by default this tool only gives you the first 10-15 minutes of chat logs.  You have to type "-l 0" if you want chat logs for the entire length of the video.  Replace "XXXXXXXXXX" with the video ID number that you will get from the URL bar in your browser once you've loaded the desired video.

So you're welcome fellow Twitch Chatlog user.  No thanks to "freaktechnik".  Don't bother replying to me.  because I used a dummy email account to type these Instructions and I won't be getting any email notifications about your replies.  And I'm banned on my main gaveitatry02 account, so I can't respond  to you on there.  Just "good luck" and "take care".  Adios.
